src/main/tools/jdk.BUILD

```diff
@@ -181,6 +181,12 @@ filegroup(
     ),
 )
 
+#This folder holds security policies
+filegroup(
+    name = "jdk-conf",
+    srcs = glob(["conf/**"]),
```

Comment:
The `conf` folder holds security policy files (`limited` and `unlimited`). We use a Java mock-server that in tests that uses an untrusted cert, so we rely on `unlimited` policy to allow this to work.

Reply:
we use Corretto JDK, and it has about 20 files under that folder (and its sub-folders) around 100kb in size.